libclc/clc/lib/generic/conversion/clc_convert_int2float.cl

```diff
@@ -0,0 +1,103 @@
+//===----------------------------------------------------------------------===//
```

Comment:
This file is not a source, it is only included. Should it have the `.inc` extension?